README.rst

```diff
@@ -45,21 +48,32 @@ Getting Started
 
       $ make test.run_elasticsearch
 
-3. Create a user and authentication token. Note that the user will be
+#. Create a user and authentication token. Note that the user will be
    created if one does not exist.
 
    ::
 
        $ ./manage.py set_api_key <username> <token>
```

Comment:
I always wondered what this token here should be, maybe can add a line? I have been entering garbage values for it like '1alkdjflsd'

Reply:
Ha, I'm not exactly sure what it is myself; I assume it's a token for API access? imho we should be generating that ourselves and giving it to the user, not the other way around...